test/test.go

```diff
@@ -506,12 +506,7 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
-	// Ensure it's the absolute, symlink-less path we're returning
-	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
Would be good to have a bit of a shove in the right direction with how to choose to fix this one up.